exampleContracts/exampleContracts.test.ts

```diff
@@ -1,10 +1,49 @@
 import { describe } from 'mocha';
-import { runTests } from '../tests/testing';
+import { ResultType, runTests } from '../tests/testing';
 
-const TIME_LIMIT = 2 * 60 * 60 * 1000;
+import path from 'path';
 
-// Transpiling the solidity files using the `bin/warp transpile` CLI command.
-describe('Transpiling example contracts:', function () {
-  this.timeout(TIME_LIMIT);
-  runTests(process.argv.includes('--force', 3) ? true : false, false);
-});
+const WARP_TEST = 'warpTest';
+const WARP_COMPILATION_TEST_PATH = 'exampleContracts';
+const WARP_TEST_FOLDER = path.join(WARP_TEST, WARP_COMPILATION_TEST_PATH);
+
+const expectedResults = new Map<string, ResultType>(
+  [
+    ['nestedStructs.sol', 'Success'],
```

Comment:
Is it possible to organize this list alphabetically? It's just so it would be easier to find the contracts that way. It's also the way we do it in other expectations files